doc/index.rst

```diff
@@ -136,6 +154,7 @@ What this version provides
   * :doc:`2d_combine`
   * :doc:`coadd1d`
   * :doc:`coadd2d`
+  * :doc:`known_failure_modes`
```

Comment:
I think it would be better if we put this in one of the toc trees below so that it shows up in the left sidebar.

Reply:
ah, that was my intent.  

Have done so now.